config.go

```diff
@@ -39,6 +39,11 @@ type Config struct {
 	// LineLength prevents descriptions and unreasonably long lines. Can be disabled
 	// completely by setting it to 0.
 	LineLength uint
+
+	// rootSchema is used to keep track of the root of the schema we're traversing, in case
+	// we want to use lookups to find a $ref. Not exported as users aren't meant to tinker
+	// with this value yet.
+	rootSchema *JSONSchema
```

Comment:
a $ref can also follow a filepath (file://) which is relative to some dir, how would such a ref be evaluated?

Reply:
Both remote and file references are currently not in scope for this change, and I might never add those as I don't have a use-case for it